tests/management/commands/create_fixtures.py

```diff
@@ -0,0 +1,49 @@
+"""Create serialized data for tests and store this data in a json file."""
+import os
+from contextlib import contextmanager
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+from pages.models import CustomPage
+from tests.catalog import models as catalog_models
+
+
+class Command(BaseCommand):
```

Comment:
no method of this class uses `self`. So, it's not class, but container with incohesive methods.
Maybe we should create some `DataGenerator` or `PagesGenerator` with more relevant methods?